src/page/summary-step.page.ts

```diff
@@ -1,20 +1,22 @@
 import { $, ElementFinder } from 'protractor';
 
 export class SummaryStepPage {
+
   private confirmOrderButton: ElementFinder;
+  private message: ElementFinder;
 
   constructor () {
     this.confirmOrderButton = $('#cart_navigation > button > span');
+    this.message = $('#center_column > div > p > strong');
   }
 
   public async confirmOrder(): Promise<void> {
     await this.confirmOrderButton.click();
   }
 
   public async mensajeText(): Promise<string> {
-    const mensaje = $('#center_column > div > p > strong').getText();
-    return mensaje;
-
+    const message = this.message.getText();
+    return message;
```

Comment:
Podrías ahorrarte una línea de código si solo le colocar 'return this.message.getText()' directamente y recuerda ponerle el await para esperar la promesa.

Reply:
corregido en summary-step.page